File: src/app/todo.ts
export class Todo {
    private isActive: boolean = true;
    private description: string;
    private id: number = Math.random();

    constructor(isActive: boolean, description: string) {
        this.description = description;
        this.isActive = isActive;
    }

    setIsActive(isActive: boolean): void {
        this.isActive = isActive;
    }

    getIsActive(): boolean {
        return this.isActive;
    }

    setDescription(description: string): void {
        this.description = description;
    }

    getDescription(): string {
        return this.description;
    }

    getId(): number{
        return this.id;
    }
}

File: src/main/java/com/todo/todo/DAO/ToDo.java
package com.todo.todo.DAO;

import jakarta.persistence.Column;

public class ToDo {
    private Integer id;
    private String description;
    private boolean active;

    public ToDo() {
    }

    public ToDo(Integer id, String description, boolean isActive) {
        this.id = id;
        this.description = description;
        this.active = isActive;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public boolean getActive() {
        return active;
    }

    public void setActive(boolean active) {
        this.active = active;
    }
}
